eZ/Bundle/EzPublishCoreBundle/Resources/views/default/full.html.twig

```diff
@@ -0,0 +1,5 @@
+<h2>{{ ez_content_name(content) }}</h2>
+{% for field in content.FieldsByLanguage %}
```

Comment:
PhpStorm displays the full method name in autocomplete dropdown but when you select the method, it autocompletes to lowercase. So for example, it's not `ezpublish.configResolver.Parameter`, but `ezpublish.configResolver.parameter`